### DISCUSSION TITLE
[ask] to change network images

### DISCUSSION THREAD

**Discussion by ebennaga** [WalletConnect/walletconnect-monorepo#3551] [Developer Support]
i have been read documentations to change custom network images but doesn't work. what should i do ?

this is my code
` const chainImages: any = [
        {
            1399811149: 'images/chest/gold.png',
            122: 'images/chest/solana.webp'
        }
<Web3Modal projectId={projectId} ethereumClient={ethereumClient} tokenImages={chainImages} chainImages={chainImages} />`

**Comment by glitch-txs** [comment]
hi, could you share a [MRE](https://github.com/orgs/WalletConnect/discussions/3457)?
